### PR TITLE
Pic 213 get case list endpoint

### PR DESCRIPTION
**Pull request by jhevans**
Implemented endpoint to bring back all cases with session time on a given date

### CODE REVIEW

File: src/test/resources/before-test.sql [hunk 1]
```diff
@@ -4,5 +4,12 @@ DELETE FROM courtcaseservicetest.court WHERE true;
...
 
+-- See CourtCaseControllerTest.shouldGetCaseListWhenCasesExist()
+-- These records are used to test edge cases when returning court case list for a given date (midnight to 1 second before midnight the next day)
```
Comment (katie-cawthorne) [inline src/test/resources/before-test.sql]: Sorry due to my recent database changes, this will need amending.

Comment (jhevans) [reply src/test/resources/before-test.sql]: No problem, updated and all tests passing
